grommet-leaflet/src/themes/base.d.ts

```diff
@@ -0,0 +1,23 @@
+import { BoxProps, TextProps } from 'grommet';
+
+export interface LeafletThemeType {
+  pin?: {
+    [x: string]: {
+      container?: BoxProps;
+    };
+  };
+  cluster?: {
+    size?: {
+      [x: string]: {
+        container?: BoxProps;
+        label?: TextProps;
+      };
+    };
+    [x: string]:
```

Comment:
Is this allowed? Makes it clearer what this section is
```suggestion
    [kind: string]:
```

Reply:
done